src/commands/sbom/language.ts

```diff
@@ -6,7 +6,7 @@ export const getLanguageFromComponent = (component: any): DependencyLanguage | u
   const componentName = component['name']
 
   if (component['bom-ref']) {
-    if (component['bom-ref'].includes('pkg:npm')) {
+    if (component['bom-ref'].includes('pkg:npm') || component['purl'].includes('pkg:npm')) {
```

Comment:
`component['purl']` can be undefined here. We should check this is defined.

Reply:
it is not checked in the rest of the cases, should I fix those as well?

Reply:
Wow, this is a good point 😅 we should have checked then.

Do not worry for now, let's keep it like this - if we have an issue with this key, we will add a check later.